### PR TITLE
Add additional options to alarm_control_panel more info dialog

### PR DESCRIPTION
**Pull request by schmittx**
- Add additional buttons for "Arm Night" and "Arm Custom Bypass" options to more info dialog on `alarm_control_panel` entities.
- Update capitalization of state descriptions for `armed_home`, `armed_away`, `armed_night`, and `armed_custom_bypass`.

screen shot 2017-12-28 at 4 23 10 pm

### CODE REVIEW

**Comment by c727** [comment]
What about arm custom bypass?

**Comment by schmittx** [comment]
Good suggestion.

Four buttons won't fit on there if you include "Arm" in the button text (i.e. "Arm Home", "Arm Away", etc.).

Option #1 = Don't include "Arm Custom Bypass"
Option #2 = Remove "Arm" from each button, producing "Home", "Away", "Night", and "Custom Bypass". Not sure if that would be confusing though?
Option #3 = Use some other interaction method (i.e. drop-down list) rather than buttons

Any thoughts?

**Comment by MartinHjelmare** [comment]
I say drop-down list. But I'd like to hear someone else's opinion too.

**Comment by schmittx** [comment]
Disregard my earlier comment.

Not sure what I had tested earlier, but it is possible to fit everything in using buttons (see screenshot).

PR is updated and commit pushed.

**Comment by schmittx** [comment]
@c727 @MartinHjelmare Any concerns with this latest commit? Can we get this merged?

**Comment by MartinHjelmare** [comment]
A frontend dev should look at this.

**Comment by ciotlosm** [comment]
@schmittx 
Using buttons like that is hard to capture a correct layout especially with multi-language in mind. Easiest to start with is a dropdown as suggested, until someone with some UI alignment skills could suggest something else. 

As mentioned on a different PR the column can get very narrow on smaller screens and even in their current state the buttons won't fit.

File: src/translations/en.json [hunk 1]
```diff
@@ -61,10 +61,10 @@
     "alarm_control_panel": {
       "armed": "Armed",
       "disarmed": "Disarmed",
-      "armed_home": "Armed home",
-      "armed_away": "Armed away",
-      "armed_night": "Armed night",
-      "armed_custom_bypass": "Armed custom bypass",
+      "armed_home": "Armed Home",
```
Comment (c727) [inline src/translations/en.json]: old style is correct: https://material.io/guidelines/style/writing.html#writing-capitalization-punctuation

Comment (schmittx) [reply src/translations/en.json]: Ugh, it looks weird to me but I understand. Gotta follow the rules I suppose, I'll update with another commit.

**Comment by schmittx** [comment]
@ciotlosm Understood, I appreciate the feedback. I'll work on a layout for a dropdown method but what about using icons on the button rather than text (i.e. bell, bell-off, bell-plus, bell-sleep, etc.)? It'd eliminate the multi-language concerns as long as it'd be clearly understood what each button does. Is there any precedent for this type of UI?

**Comment by hawk259** [comment]
Would be nice to be able to add other buttons as well.  AlarmDecoder has alarmdecoder_alarm_toggle_chime service that would be nice to add a button/option for.

May be add a option to just send input directly to the alarm.  This would allow doing just about anything you want (<code><options>).

**Comment by arsaboo** [comment]
Agree with @raccettura...dropdowns require multiple clicks. Also, while we are at it, would be nice if we can give them a "button" feel.

**Comment by c727** [comment]
Let's try a 2x2 grid with raised buttons. This is a dialog with many mixed content so it should be ok

**Comment by schmittx** [comment]
@c727 Sounds good to me. Do you know of any existing components with a similar layout that I could reference? I'm still new to HTML...

**Comment by c727** [comment]
https://www.webcomponents.org/element/PolymerElements/paper-button

**Comment by c727** [comment]
If you want to continue this PR you should rebase it on latest changes

**Pull request closed without merging**